actors/builtin/power/power_actor.go

```diff
@@ -259,10 +260,15 @@ func (a Actor) UpdatePledgeTotal(rt Runtime, pledgeDelta *abi.TokenAmount) *abi.
 // This number is empirically determined
 const GasOnSubmitVerifySeal = 34721049
```

Comment:
Not a big deal, but you may want to make this point to the constant in the V0 repo the way we do parameter types.